src/js/app.js

```diff
@@ -775,7 +775,12 @@ requirejs(['jquery','app/storage','knockout','knockout-secure-binding','hjls','a
       } 
       Resting.tabContexts.remove(tab);
     };
-    
+
+   const disableSaveButton = (value) => {
```

Comment:
Is this method used ?

Reply:
@mirkoperillo No it's not used. I will remove it.